crypto/crypto.py

```diff
@@ -585,7 +597,66 @@ def __init__(
         self.childNum = childNum
         self.isPrivate = isPrivate
 
-    def deriveCoinTypeKey(self, coinType):
+    @staticmethod
+    def new(seed):
```

Comment:
`new` was the function `newMaster` before. Just moved it here as a static method.